File: klaxonModels/None.kt
package io.homemetrics.homemetrics.schemas

import com.beust.klaxon.Klaxon

data class None(
	val time_stamp: Integer?,
	val keys: Integer?,
	val unread_notification_count: Integer?,
	val house_status: Integer?,
	val flow_graph: Integer?,
	val pressure_graph_avg: Integer?,
	val pressure_graph_low: Integer?,
	val pressure_graph_high: Integer?,
	val temp_graph_avg: Integer?,
	val temp_graph_low: Integer?,
	val temp_graph_high: Integer?
)

File: swiftSchema/None.swift
import Foundation

struct None: Codable, Identifiable, Equatable {
	var id = UUID()

	var time_stamp: Int?
	var keys: Int?
	var unread_notification_count: Int?
	var house_status: Int?
	var flow_graph: Int?
	var pressure_graph_avg: Int?
	var pressure_graph_low: Int?
	var pressure_graph_high: Int?
	var temp_graph_avg: Int?
	var temp_graph_low: Int?
	var temp_graph_high: Int?

	private enum CodingKeys: String, CodingKey {
		case time_stamp, keys, unread_notification_count, house_status, flow_graph, pressure_graph_avg, pressure_graph_low, pressure_graph_high, temp_graph_avg, temp_graph_low, temp_graph_high
	}
}